synapse/lib/stormtypes.py

```diff
@@ -1705,8 +1724,22 @@ def __init__(self, runt, ldef, path=None):
             'get': self._methLayerGet,
             'pack': self._methLayerPack,
             'repr': self._methLayerRepr,
+            'edits': self._methLayerEdits,
```

Comment:
This is not safe for httpapi callers to use

Reply:
yep, ack.